### DISCUSSION TITLE
BingAI-Client 1.7.1 补丁+更新

### DISCUSSION THREAD

**Discussion by xbzstudio** [xbzstudio/Timeless-Sydney#58] [建议]
1，修改了小部分内容
2，废除phone.html，将index.html改成可以兼容手机的页面
3，添加QuickSetup.py，可以进行快速配置BingAI-Client。
（感谢XiaoJiang0208！）

<hr /><em>This discussion was created from the release <a href='https://github.com/xbzstudio/BingAI-Client/releases/tag/BingAI-Client1.7.1(zhengshiban)'>BingAI-Client 1.7.1 补丁+更新</a>.</em>

**Comment by qianchuibailian** [comment]
我换了人格设置的语言，结果出现错误了，设置失败

**Reply by xbzstudio** [reply]
什么意思？在哪设置人格？

**Reply by qianchuibailian** [reply]
就是设置Sydney的傲娇人格的时候啊，不是要输入一段话的嘛，我就改了一下语言，然后打开显示的是chat500错误。还有刚才我用的时候用到一半也是这个错误，然后他好像就忘了之前的内容了

**Reply by 11407799** [reply]
不能显示图片
要求显示图片的时候
Chat Error: message: 未知错误; http500.

**Reply by hankeigi** [reply]
> 不能显示图片 要求显示图片的时候 聊天错误： 消息： 未知错误;http500.

我也是。。。

**Reply by 11407799** [reply]
这几天突然上不去了。不知大家是不是也这样？

**Reply by chen000318** [reply]
今天突然报了这个错误：httpx.ConnectError: [Errno 0] Error
是哪里出错了?
